### PR TITLE
Adding 'UpdateSolutionDescription' Task

### PR DESCRIPTION
**Pull request by killerrin**
I required a variation of the changes in #14 so I figured I would submit a PR. I tried to make it generalized by giving some options for how people want to apply their new descriptions.

Let me know if there are any issues

### CODE REVIEW

**Comment by WaelHamze** [comment]
@killerrin 

First thanks for this. I'll add it to the toolkit. Out of interest, How do you intend to use this task?

**Comment by killerrin** [comment]
Awesome, thanks.

For us, we wanted to use it to provide an easy method of keeping track of which person on our team had completed a given build of our solution, when that build was completed, what version it was coming from as well as appending in some additional information such as a summarized change-log (from a .txt file) which is read in using a separate Powershell script and saved to an environment variable for usage in this script.

Basically, a bunch of convenience things so that developers are able to see all the information they need at the solution level instead of needing to go search through TFS or our shared file directories.

**Comment by WaelHamze** [comment]
@killerrin thanks for sharing.
I had to make some minor changes to get this to work in the pipeline. Great work though.
One observation, When appending to top or bottom it doesn't seem to be adding line breaks so its more like appending to beginning or end. I tried adding the line breaks to the description field but that didn't work. Maybe need to do this in the powershell script.

**Comment by killerrin** [comment]
Yeah, I wasn't sure how I wanted to go about the AppendToTop and AppendToBottom features. Whether I should have enforced the setting of the new line automatically in the PS Script or require the developer to add them in with a \n linebreak.

I ultimately went with them needing to be explicitly stated by the developer, which worked with how I had my personal version setup by having the description be dynamically generated. In hindsight though, It probably would have made more sense to have them as an additional checkbox for if they wanted an automatic newline.

**Comment by karan-mittal** [comment]
There is a limitation in the description field of the Solution information that it cannot accept more than 2000 characters.
Better to go with the 'Replace' option as Description Update Method.

_A validation error occurred. The length of the 'description' attribute of the 'solution' entity exceeded the maximum allowed length of '2000'._
